resources/views/patients/edit.blade.php

```diff
@@ -1,160 +1,95 @@
-                            <h5>Follow Us</h5>
-                            <ul class="list-unstyled d-flex align-items-center justify-content-center mt-3">
-                                <li><i class="fa-brands fa-facebook me-3"></i></li>
-                                <li><i class="fa-brands fa-instagram me-3"></i></li>
-                                <li><i class="fa-brands fa-twitter"></i></li>
-                            </ul>
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="phone" class="form-label">Phone Number *</label>
+                            <input type="text" class="form-control" id="phone" name="phone"
+                                placeholder="09*********" value="{{ $patient->phone }}">
                         </div>
-                        <div
-                            class="col-lg-4 col-md-6 col-sm-12 text-lg-center text-md-start text-sm-center mt-md-3 mt-sm-3">
-                            <h5>Support Us</h5>
-                            <ul class="list-unstyled mt-3">
-                                <li><a href="#" class="text-decoration-none text-light">Documentation</a></li>
-                                <li><a href="#" class="text-decoration-none text-light">FAQS</a></li>
-                                <li><a href="#" class="text-decoration-none text-light">About Us</a></li>
-                                <li><a href="#" class="text-decoration-none text-light">Contact Us</a></li>
-                            </ul>
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="age" class="form-label">Age *</label>
+                            <input type="number" class="form-control" id="age" name="age" placeholder="age"
+                                value="{{ $patient->age }}">
                         </div>
-
                     </div>
-                    <div class="row mt-3 mb-4">
-                        <div class="col-12 text-sm-center">
-                            <small class="text-light">2023 &copy; CLINIC.ORG &trade;</small>
+                    <div class="col-lg-6 col-md-6">
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="address" class="form-label">Address *</label>
+                            <input type="text" class="form-control" id="address" name="address" placeholder="address"
+                                value="{{ $patient->address }}">
+                        </div>
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="blood_type" class="form-label">Blood Type</label>
+                            <select class="form-select" aria-label="Default select example" id="blood_type"
+                                name="blood_type">
+                                <option>Select Blood Type</option>
```

Comment:
What for this option field?
`<option {{ $patient->blood_type == null ? 'selected' : '' }}  value="0">Select Blood Type</option>`